.github/CODEOWNERS

```diff
@@ -101,7 +101,7 @@ clang/tools/clang-offload-extract/ @sndmitriev @mlychkov @AlexeySachkov
 # Explicit SIMD
 SYCLLowerIR/ @kbobrovs @DenisBakhvalov
 esimd/ @kbobrovs @DenisBakhvalov
-sycl/include/sycl/ext/intel/experimental/esimd.hpp @kbobrovs @DenisBakhvalov
+sycl/include/sycl/ext/intel/experimental/esimd/ @kbobrovs @DenisBakhvalov @kychendev
```

Comment:
esimd directory is already covered by previous rule, so you can just add @kychendev to the line above.
Also, shouldn't we add all three accounts too all 4 patterns?

Reply:
Ah, OK, I didn't know that `esimd/` was covered. Added us to all ESIMD-related sources.

Reply:
Thanks!